### PR TITLE
Add defaultTTL option relative to cell write timestamps

### PR DESCRIPTION
**Pull request by dkropachev**
## Summary
- Adds a `defaultTTL` config option for Cassandra/Scylla sources that computes an effective TTL relative to each cell's write timestamp (`defaultTTL - age`), so data expires at the correct absolute time in the target
- Validates at config load time: must be non-negative, fit in 32 bits, require `preserveTimestamps: true`, and a Scylla target
- Includes deterministic unit tests for `computeEffectiveTTL` with injectable clock

## Test plan
- [x] `ComputeEffectiveTTLTest` — 8 unit tests covering: existing TTL preserved, no TTL/no default returns 0, relative TTL computation, expired records get TTL=1, missing writetime returns 0, boundary cases
- [x] Manual test with a Cassandra source table containing cells with and without TTLs
- [x] Verify config rejects negative `defaultTTL`, values > `Int.MaxValue`, non-Scylla targets, and `preserveTimestamps: false`

### CODE REVIEW

File: docs/source/configuration.rst [hunk 1]
```diff
@@ -270,6 +270,13 @@ Apache Cassandra Target
     writeTTLInS: 7776000
     # Optional - writetime in microseconds (sample 1640998861000 is Saturday, January 1, 2022 2:01:01 AM GMT+01:00 )
     writeWritetimestampInuS: 1640998861000
+    # Optional TTL (in seconds) applied relative to each cell's write timestamp.
+    # Requires source preserveTimestamps: true. Only use when migrating into an empty target table.
+    # Accepts a plain number (defaults to set-if-missing) or "value:policy":
```
[the remark this answers is not in the document]
Comment (tarzanek) [reply docs/source/configuration.rst]: (and config.yaml.example should really mirror code 🙈 )

**Comment by tarzanek** [comment]
the parquet test failure is related to this PR? I'd doubt it, rerunning
